woocommerce/woocommerce
# Images Product management MVP 1.0

### All Submissions:






### Changes proposed in this Pull Request:



This PR adds the Image section to the product creation/edition.

![Screen Capture on 2022-09-22 at 17-42-32](https://user-images.githubusercontent.com/1314156/191847485-587573a1-3690-409f-b37f-0d2c0773d4f3.gif)

Detail:
- It adds an Images section.
- It adds the upload image component and a gallery to see and order the images.

The following functionalities are not included here, but they will be added in a follow-up PR.
- Clicking anywhere on the card (except for buttons) opens the file picker.
- The addition of a toolbar from the Gallery block in Gutenberg to manage gallery images.

Closes #34759.

### How to test the changes in this Pull Request:

1. Checkout this branch.
2. Grab one (or more) image from your computer and drop it in the `Drag images here or click to upload` area.
3. Verify that the image has been uploaded correctly and is now being shown on the screen.
4. Verify that the `Drag images here or click to upload` area is smaller when there is at least one image loaded.
5. Upload another image and change the order of the images.
6. Then save the product. Verify that the new order for the images is saved correctly.
7. Press `Choose images` and select an image that has been uploaded before.
8. Verify that the selected image is added to the image gallery component.
9. Grab an image from the gallery and verify that the area `Drop here to remove` is now visible.
10. Drop the image in that area and verify that the image is removed from the gallery.

### Other information:




### FOR PR REVIEWER ONLY:
